### PR TITLE
fix autorenew validation and E2E that failing because of that

### PR DESCRIPTION
**Pull request by povolev15**
Fix the issue that some of the E2E was failing related to the autorenew validation in file update.

### CODE REVIEW

File: hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileUpdateTest.java [hunk 1]
```diff
@@ -286,7 +285,6 @@ void validatesNewMemo() {
     }
 
     @Test
-    @Disabled
     void validatesExpirationTimeIsNotInRange() {
```
Comment (jsync-swirlds) [inline hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileUpdateTest.java:288]: Perhaps should be `validatesAutoRenewDurationIsInRange`

File: hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileUpdateTest.java [hunk 2]
```diff
@@ -407,34 +411,46 @@ void appliesNewContentUpgradeFile() {
...
+                .build();
         when(handleContext.body()).thenReturn(txBody);
         given(handleContext.verificationFor(Mockito.any(Key.class))).willReturn(signatureVerification);
         given(signatureVerification.failed()).willReturn(false);
 
         subject.handle(handleContext);
 
         final var newFile = writableFileState.get(fileId);
-        assertEquals(1_234_567L, newFile.expirationSecond());
+        assertEquals(3_456_789L, newFile.expirationSecond());
```
Comment (jsync-swirlds) [inline hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileUpdateTest.java:453]: This number appears a few times in the file.  Might be worth introducing a constant for it.

**Review by vtronkov — APPROVED** [review]
Other than Joseph's comments, LGTM

File: hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileUpdateHandler.java [hunk 3]
```diff
@@ -189,19 +188,21 @@ private void resolveMutableBuilderAttributes(
...
-                    .getConfigData(EntitiesConfig.class)
-                    .maxLifetime();
-            final var now = handleContext.consensusNow().getEpochSecond();
-            final var expiryGivenMaxLifetime = now + maxEntityLifetime;
+            final long effectiveDuration = op.expirationTime().seconds()
+                    - handleContext
+                            .body()
+                            .transactionID()
+                            .transactionValidStart()
+                            .seconds();
```
[the remark this answers is not in the document]
Comment (povolev15) [reply hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileUpdateHandler.java:198]: done thanks

File: hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileUpdateTest.java (hunk 2 above)
Comment (povolev15) [reply hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileUpdateTest.java]: done thanks

**Review by vtronkov — APPROVED** [review]

**Review by jsync-swirlds — APPROVED** [review]

**Review by kimbor — APPROVED** [review]
lgtm!

**Pull request merged**